.github/workflows/ci.yml

```diff
@@ -7,13 +7,7 @@ on:
 
 jobs:
   build-and-test:
-    runs-on: ${{ matrix.os }}
-    env:
-      USE_BAZEL_VERSION: ${{ matrix.bazel_version }}
-    strategy:
-      matrix:
-        os: [ubuntu-20.04]
-        bazel_version: [bazelbuild/5.4.1, bazelbuild/6.4.0]
+    runs-on: ubuntu-20.04
```

Comment:
While you're here could we bump this to 24.04? 20.04 is going away soon on GitHub actions.

Reply:
Done!